packages/nextjs/pages/example-zk/SignedStats.tsx

```diff
@@ -0,0 +1,128 @@
+import type { NextPage } from "next";
+import { useBirthYearProofsStore } from "~~/services/store/birth-year-proofs";
+
+function shortenHashString(hash: string) {
+  const prefix = hash.substring(0, 2);
+  const suffix = hash.substring(hash.length - 4);
+
+  return prefix + "..." + suffix;
+}
+
+export function copyToClipboard({
```

Comment:
Is not better to use the same tool as they use here? `packages/nextjs/components/scaffold-eth/Address.tsx` (`react-copy-to-clipboard`). I can change to that in my PR instead if you want.

Reply:
i agree, changed this